code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -43,6 +43,13 @@
 		"bromine",
 		"stable_plasma"
 	)
+	var/list/emagged_reagents = list(
+		"space_drugs",
+		"morphine",
+		"carpotoxin",
+		"regenerative_materia",
```

Comment:
please, please, please do not put blob chems in anything but the blob, they're not meant to ever be analyzed or messed with